test/rubygems/test_gem_spec_fetcher.rb

```diff
@@ -168,20 +168,20 @@ def src.fetch_spec(name)
   def test_suggest_gems_from_name_latest
     spec_fetcher do|fetcher|
       fetcher.spec "example", 1
-      fetcher.spec "other-example", 1
+      fetcher.spec "an-example", 1
       fetcher.spec "examp", 1
     end
 
     suggestions = @sf.suggest_gems_from_name("examplw", :latest, 1)
     assert_equal ["example"], suggestions
 
-    suggestions = @sf.suggest_gems_from_name("other")
-    assert_equal ["other-example"], suggestions
+    suggestions = @sf.suggest_gems_from_name("anexample")
+    assert_equal ["an-example"], suggestions
 
-    suggestions = @sf.suggest_gems_from_name("exam")
+    suggestions = @sf.suggest_gems_from_name("xample")
     assert suggestions.any? { ["examp"] }
     assert suggestions.any? { ["example"] }
-    assert suggestions.any? { ["other-example"] }
+    assert suggestions.any? { ["an-example"] }
```

Comment:
These tests only check if suggestions is not empty. Maybe this was supposed to be assert suggestions.include?("examp")